fuzz/fuzz_targets/transaction_check_outputs.rs

```diff
@@ -0,0 +1,12 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use monero::util::test_utils::{
+    fuzz_create_transaction_1,
```

Comment:
`fuzz_create_transaction_1` has been renamed right? it and `fuzz_create_transaction_2` are still used in some places

Reply:
Yip, thanks for picking that up. The fuzzing tests are not compiled unless one runs them.